Library/Formula/icecream.rb

```diff
@@ -0,0 +1,97 @@
+class Icecream < Formula
+  homepage "https://en.opensuse.org/Icecream"
+
+  option "without-docbook2X", "Build without man page"
+  option "enable-clang-wrappers", "Use symlink wrappers for clang/clang++"
+  option "enable-clang-rewrite-includes", "Use by default Clang's -frewrite-includes option"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "docbook2X" => [:recommended, :build]
```

Comment:
Should probably make this `:optional` given it requires manual intervention to install a perl lib.